### ISSUE TITLE
Check course overview rendering w.r.t. rubric rows

### ISSUE THREAD

**Issue by jadudm** [CMSgov/CMCS-DSG-DSS-Oversight#239]
Related to previous issue; this may have been resolved, but double-check.

**Comment by jadudm** [comment]
I have two thoughts here:

1. The rubric rows will not render well on small screens. They need CSS fallback to plain text.
2. The DIV rows may not read well on screen readers. Therefore... perhaps the rows should be replaced with their textual versions.
3. Every lesson should, perhaps, have a standard header that is drawn from the rubric and metadata at the top of the lesson.

This last point would take some work, as it involves touching every lesson. But, it would clean the site up tremendously in terms of how every lesson starts out.

**Comment by jadudm** [comment]
I've created a standard template called "lesson," and it applied nicely to the first course. It's work applying to all of the rubric lessons.

Not sure about the DIV concerns, in the end. I spent some time noodling on this, and either 1) some kind of JS/cookie solution is needed, or 2) a dual-render on pages, where one version has the text-based version, and the other has the DIVs. 

I do need to think a bit more about ARIA elements and accessibility, but truth told, we're doing pretty good as-is.
